hawc/apps/assessment/api.py

```diff
@@ -165,6 +165,14 @@ class Assessment(AssessmentViewset):
     serializer_class = serializers.AssessmentSerializer
 
 
+class PublicAssessment(viewsets.ReadOnlyModelViewSet):
```

Comment:
add a new `@action(detail=False, methods=('get',))` view to `AssessmentViewset` instead of creating a new viewset

Reply:
`AssessmentViewset` acts a parent class for several other viewsets, so I added this `@action` to the `Assessment` viewset instead.